test/spec.sh

```diff
@@ -709,6 +709,11 @@ extglob-match() {
     $BASH $MKSH $OSH_LIST "$@"
 }
 
+nocasematch-match() {
+  sh-spec spec/nocasematch-match.test.sh \
```

Comment:
Please `add --osh-failures-allowed 6` here.  That just indicates "we know about these failures, the whole test suite shouldn't fail"

Reply:
Just to clarify, it only affects `case` and `[[` and the rest are to verify that it does not affect it.  As such, OSH currently passes 3 of the tests, so should it be `--osh-feailures-allowed 3`?

Reply:
Ah yes, it should be 3.  You can leave the other tests since the look informative.